crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -0,0 +1,41 @@
+#    Copyright (C) 2020 Compassion CH
+#    @author: Quentin Gigon
+
+from odoo import models, api, fields
+
+
+class CrowdfundingParticipant(models.Model):
+    _name = "crowdfunding.participant"
+    _description = "Participant to one of our crowd-fundings"
+
+    project_id = fields.Many2one("crowdfunding.project", string="Partner project")
+    partner_id = fields.Many2one("res.partner", string="Partner")
+    personal_motivation = fields.Text(string="Personal Motivation")
+    product_number_goal = fields.Integer()
+    product_number_reached = fields.Integer(compute="_compute_product_number_reached")
+    number_sponsorships_goal = fields.Integer()
+    number_sponsorships_reached = fields.Integer(
+        compute="_compute_number_sponsorships_reached")
+    # TODO fix one2many fields
+    sponsorship_ids = fields.One2many(
+        "recurring.contract",
+        "group_id",
```

Comment:
This would be equivalent to the counterpart field name, here I propose `crowdfunding_participant_id` as suggested in the comment above